.travis.yml

```diff
@@ -367,6 +379,11 @@ script:
     - export OSHRUN_LAUNCHER="$SOS_PM"
     - cd $TRAVIS_SRC/tests-uh
     - make $TRAVIS_PAR_MAKE C_feature_tests
+    - >
+      if [[ $SOS_BUILD_OPTS = *"with-pmix"* ]]; then
+          sed -i '/RUN_CMD=\"oshrun\"/c\our $RUN_CMD=\"\";' common/testrunner.pl
+          export RUN_OPTIONS=$SOS_PM
+      fi
     - $SOS_PM_PRE
```

Comment:
@davidozog We have already disabled external tests for pmix, right? UH tests should not even run.

Reply:
Oh yeah, good call.  This is what's needed to get `prun` working with `tests-uh`, but there just isn't enough time to get through all the external tests so I ended up taking it out.  I'll revert this so it's at least in our commit history if we need it later.